specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/compute.json

```diff
@@ -8321,6 +8970,253 @@
       ],
       "description": "The list dedicated host operation response."
     },
+    "CapacityReservationGroupInstanceView": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
```

Comment:
missed capitalization

Reply:
This is a property name. So followed camel casing like all other properties.